### ISSUE TITLE
Error After Installing nuxt-rive: Missing Export for 'Rive' and 'EventType'

### ISSUE THREAD

**Issue by norbertSG** [mrnasil/nuxt-rive#2]
Nuxt 3 version 3.10.1
I run the `npm run --save-dev nuxt-rive'

nuxt.js?v=dcc59c25:98 [nuxt] error caught during app initialization SyntaxError: The requested module '/_nuxt/node_modules/@rive-app/webgl/rive.js?v=dcc59c25' does not provide an export named 'Rive' (at index.mjs?v=dcc59c25:79:15)Caused by: SyntaxError: The requested module '/_nuxt/node_modules/@rive-app/webgl/rive.js?v=dcc59c25' does not provide an export named 'Rive' (at NuxtRive.client.vue:15:10)

**Comment by mrnasil** [comment]
You’re right. I’m aware of the mistake. I’ll release an update at the earliest opportunity. I’m not satisfied with the structure. My apologies.

**Comment by norbertSG** [comment]
But, how is working here https://stackblitz.com/~/github.com/mrnasil/nuxt-rive?file=package.json:L38 ?

**Comment by norbertSG** [comment]
@mrnasil Do you an eta for the fix ?

**Comment by suchorski** [comment]
Waiting for the answer too

**Comment by suchorski** [comment]
> But, how is working here https://stackblitz.com/~/github.com/mrnasil/nuxt-rive?file=package.json:L38 ?

perhaps add '@rive-app/webgl' to dependencies

**Comment by mrnasil** [comment]
> [@mrnasil](https://github.com/mrnasil) Do you an eta for the fix ?

I apologize for the delay. I would appreciate it if you could try the new version and share your experience. I hope the issue has been resolved.
